pageTests/api/session.test.js

```diff
@@ -103,6 +106,75 @@ describe("api/session", () => {
           })
         );
       });
+      it("logs a login event when a user has logged in", async () => {
```

Comment:
it might be good to have a test around what happens when the `logEvent` gateway returns an error?